Remotion/Validation/UnitTests/TestDomain/Validators/FakeObjectValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+using System;
+using System.Collections.Generic;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.Results;
+using Remotion.Validation.Validators;
+
+namespace Remotion.Validation.UnitTests.TestDomain.Validators
+{
+  public class FakeObjectValidator : IObjectValidator
+  {
+    public ValidationMessage ValidationMessage { get; }
+
+    public FakeObjectValidator (ValidationMessage validationMessage)
+    {
+      ValidationMessage = validationMessage;
+    }
+
+    public IEnumerable<ObjectValidationFailure> Validate (ObjectValidatorContext context)
+    {
+      throw new NotImplementedException();
```

Comment:
It's okay to keep this as NotImplementedException. 

Suggestion:
I think I usually go to `return Enumreable,Empty<...>()` in cases such as this because it means the type won't break validation runs should it ever get used during such an operation.